week_1/main.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>week_1 Assignment</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <h1 style="padding-left:40px;">
+        My Guest Book
+    </h1>
+    <p>
+        <div class="first line" style="padding-left:40px;">
```

Comment:
Umm, if you are not using the first and line classes, you dont really have to add them.